app/containers/main-container.jsx

```diff
@@ -3,16 +3,46 @@ import { connect } from 'react-redux'
 
 /* components */
 import GoalsList from './../components/goals-list/goals-list.jsx';
+import AddGoal from './../components/add-new-goal/add-new-goal.jsx';
+
+/* ations */
+import action_stepAddGoal from './../actions/goals-list.js';
+
 
 const MainContent = props => {
+
+  let component;
+
+  switch (props.step.currentStep) {
+    case 'ADD_GOAL':
+      component = <AddGoal />
+      break;
+    default:
+      component = <GoalsList
+        goals={ props.goals }
+        stepAddGoal={ props.stepAddGoal }
+      />
+  }
+  const navbarHeight = 90;
+
+  const dynamicStyle = {
+    height: window.innerHeight - 90,
```

Comment:
Not important now, but you can
  1. listen to windowResize events somewhere,
  2. update the redux state by dispatching an action with the new height
  3. pass this as props into your app.

Not important now but a pattern that redux handles well. (reactive programming or something like that..)